.github/workflows/release-nym-vpn-x.yml

```diff
@@ -126,6 +126,48 @@ jobs:
           # set gh outputs
           echo "pkg_version=$version" >> "$GITHUB_OUTPUT"
 
+      # Download latest release of nym-vpnd (windows)
+      - name: Get nym-vpn-core release info
+        if: contains(matrix.os, 'windows')
+        id: get_release
+        run: |
+          $releases = Invoke-RestMethod -Uri https://api.github.com/repos/nymtech/nym-vpn-client/releases
+          $release = $releases | Where-Object { $_.tag_name -match '^nym-vpn-core-v[0-9]+\.[0-9]+\.[0-9]+$' } | Sort-Object { [version]($_.tag_name -replace 'nym-vpn-core-v', '0.') } | Select-Object -Last 1
+          $asset = $release.assets | Where-Object { $_.name -match 'windows_x86_64.zip$' }
+          Write-Output "Release Tag: $($release.tag_name)"
+          Write-Output "Asset URL: $($asset.browser_download_url)"
+          echo "::set-output name=release_tag::$($release.tag_name)"
+          echo "::set-output name=asset_url::$($asset.browser_download_url)"
+
+      - name: Download nym-vpn-core artifact
+        if: contains(matrix.os, 'windows')
+        run: |
+          $url = '${{ steps.get_release.outputs.asset_url }}'
+          $output = 'nym-vpn-core.zip'
+          Invoke-WebRequest -Uri $url -OutFile $output
+
+      - name: Unzip nym-vpn-core artifact
+        if: contains(matrix.os, 'windows')
+        run: |
+          $output = 'nym-vpn-core.zip'
+          $destination = 'nym-vpn-core-bin'
+          Expand-Archive -Path $output -DestinationPath $destination -Force
+
+      - name: List files
+        if: contains(matrix.os, 'windows')
+        run: |
+          $destination = 'nym-vpn-core-bin'
+          Get-ChildItem -Path $destination -Recurse
+
+      - name: Copy files
+        if: contains(matrix.os, 'windows')
+        run: |
+          $destination = 'nym-vpn-core-bin'
+          $subdirectory = Get-ChildItem -Path $destination | Where-Object { $_.PSIsContainer } | Select-Object -First 1
+          $sourcePath = Join-Path -Path $destination -ChildPath $subdirectory.Name
+          $targetPath = 'nym-vpn-x/src-tauri'
+          Copy-Item -Path (Join-Path -Path $sourcePath -ChildPath '*') -Destination $targetPath -Recurse -Force
```

Comment:
is it required to have those script written in powershell?
(if not you can write them as classic bash, even on windows runner, using `shell: bash`)